### ISSUE TITLE
Add comparison to Delta Chat?

### ISSUE THREAD

**Issue by andreasmjg** [simplex-chat/simplex-chat#1298]
Delta Chat is a protocol that uses existing email infrastructure (IMAP / SMTP) to deliver encrypted messages: https://delta.chat/en/ - the "global identity" would definitely still be red for Delta Chat although server federation is arguably the most mature.

To me, it doesn't appear to fit into any of the categories on the first page: https://simplex.chat/#comparison

I'm not a security expert though so perhaps it is covered?

Thanks for the good work!

Andreas

**Comment by andreasmjg** [comment]
Adding link to another comparison topic on the Session app: https://github.com/simplex-chat/simplex-chat/issues/309

**Comment by epoberezkin** [comment]
> To me, these apps don't appear to fit into any of the categories on the first page: https://simplex.chat/#comparison

Delta Chat is a "federated" design.

Briar is "P2P" design

So on the hight level they fit in the relevant columns in the table? Or do you think it's somehow outside.

> Session

We probably could add some category like "other anonymous routing", as Privacy Guides did, but Session generally fits into P2P-over-X category in the same way as Cwtch, with the difference being that Cwtch uses Tor and Session uses LokiNet, having the same downsides as all P2P designs - network-wide identifiers, single network, network-wide attacks.

**Comment by andreasmjg** [comment]
Thanks, that does make sense.

The only features of Briar that doesn't fit into your description is the WiFi & Bluetooth P2P connection, but in this case I think it's not warranted to highlight that specifically as it's a different use case. P2P Bluetooth/WiFi is for cases where peers are already in close proximity to each other and probably physical talking/shouting is possible. 

Yes you are right, Delta Chat is federated design with network-wide identifiers etc (same structure as e-mail). And these identifiers are not "laundered" after peer introduction as I understand Simplex does. So if an email address becomes "watched", all connections to that email address becomes "watched". While if I monitor connections to one Simplex address does not automatically give access to all connections to that Simplex address, not even the connections that I have observed make the connection during my watch. 

I would perhaps add a link-out to a table (like a small footnote under the table saying "Detailed Comparison with Other Secure Messaging Services" and add some more detail like this. 

I think a lot of the information is already available, e.g. here: https://github.com/simplex-chat/simplex-chat#frequently-asked-questions. 

Perhaps something to do for someone who is tired and can't work on anything complex, to put a comparison table together. Maybe I will try to make a first draft later :-)

**Comment by epoberezkin** [comment]
> The only features of Briar that doesn't fit into your description is the WiFi & Bluetooth P2P connection

Correct. We had a lively debate in some Matrix channel about P2P messengers, and while I am a big fan of mesh networks, and really hope to see somebody building a reliable mesh transport layer, I do not want to have it as part of the application. So far all mesh messengers combined transport and application layers in one, which confirms there is a demand for mesh networking, without doubt, but seems an evolutionary dead end to me... Once somebody creates a general purpose TCP mesh network (available as SOCKS5 / VPN proxy app, same as Orbot for Tor) we will make sure to make SimpleX usable via it – I really hope it won't be us to create it, but I don't see any efforts in this direction so far, which is sad...

> and not even the connections that I have observed being made during my time of watching the simplex address.

you cannot actually observe it in all cases as easily, as it doesn't necessarily happens at the same time. The way the address works is that you send the one-time link to the address already e2e encrypted, with the public key for DH exchange visible in the header.

This link can be on another server, and the recipient can only make the connection to that server at a later time. So there are several independents packets to correlate, neither of which has any identifiers or ciphertext in common even inside TLS, and also can be at a different time.

So the advantage of long term addresses is that you actually have better meta-data protection, but the downside that you do not really know who is connecting to it, you need some out-of-band authentication to confirm it, while with one-time-links you know who you shared it with but the actual server is visible in the link. Queue rotation that is shipping in 4.2 removes the latter disadvantage though, as once the queue is moved to another server it cannot be easily observed (although the rotation is likely to be lower latency).

> I would perhaps add a link-out to a table (like a small footnote under the table saying "Detailed Comparison with Other Secure Messaging Services" and add some more detail like this.

I like the idea, we may add some extended comparisons to the new website after it's launched next week.

> Maybe I will try to make a first draft later :-)

That would be very cool :)

**Comment by epoberezkin** [comment]
> Can you make queue rotation automatic (without any user action)?

that's the plan

**Comment by jermanuts** [comment]
Guess it is done?

![image](https://github.com/simplex-chat/simplex-chat/assets/109705802/548c20c8-4733-4eca-80a0-d7ff5136ab29)

![image](https://github.com/simplex-chat/simplex-chat/assets/109705802/ba12f200-6207-483e-b6e2-e07ea62b1b19)
